src/Stripe.net/Infrastructure/Public/ApiRequestor.cs

```diff
@@ -38,6 +39,23 @@ internal abstract class ApiRequestor
         /// <value>The <see cref="IHttpClient"/> used to send HTTP requests.</value>
         public virtual IHttpClient HttpClient { get; }
 
+        internal string GetBaseUrl(BaseAddress baseAddress)
+        {
+            switch (baseAddress)
+            {
+                case BaseAddress.Api:
+                    return this.ApiBase;
+                case BaseAddress.Files:
+                    return this.FilesBase;
+                case BaseAddress.Connect:
+                    return this.ConnectBase;
+                case BaseAddress.MeterEvents:
+                    return this.MeterEventsBase;
+                default:
+                    throw new ArgumentOutOfRangeException(nameof(baseAddress), baseAddress, null);
```

Comment:
`ArgumentOutOfRangeException ` seems like an odd error type to throw (I'd expect something closer to python's `ValueError`), but if it's a standard c# thing i'm fine with it

Reply:
That was there when I got here, but I do think it's right for c#: the docs for it say "The exception that is thrown when the value of an argument is outside the allowable range of values as defined by the invoked method." which applies here, I think.